schemas/dev/parser.schema.json

```diff
@@ -0,0 +1,776 @@
+          "$ref": "#/definitions/mapping"
+        },
+        "has_obesity": {
+          "description": "Obesity",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_rheumatologic_disorder": {
+          "description": "Rheumatologic disorder",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_hiv": {
+          "description": "HIV",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_hypertension": {
+          "description": "Hypertension",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_malignant_neoplasm": {
+          "description": "Malignant neoplasm",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_smoking": {
+          "description": "History of smoking",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_asthma": {
+          "description": "Asthma",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_chronic_cardiac_disease": {
+          "description": "Chronic cardiac disease",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_chronic_kidney_disease": {
+          "description": "Chronic kidney disease",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_diabetes": {
+          "description": "Diabetes",
+          "$ref": "#/definitions/mapping"
+        },
+        "diabetes_type": {
+          "description": "Diabetes type",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_liver_disease": {
+          "description": "Liver disease",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_apnoea": {
+          "description": "Apnoea",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_hiv_viral_suppression": {
+          "description": "HIV viral suppression",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_inflammatory_bowel_disease": {
+          "description": "Inflammatory bowel disease",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_rare_disease_inborn_metabolism_error": {
+          "description": "Rare disease and inborn errors of metabolism",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_solid_organ_transplant": {
+          "description": "Solid organ transplant",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_tuberculosis_past": {
+          "description": "History of tuberculosis",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_hiv_art": {
+          "description": "HIV - on ART (ARV medication)",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_immunosuppression_therapy_treatment": {
+          "description": "Immunosuppression therapies treatment",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_comorbidity_other": {
+          "description": "Other comorbidities",
+          "$ref": "#/definitions/mapping"
+        },
+        "has_died": {
```

Comment:
'has_died' will usually map to 'dsterm = 4'. i.e., true if dsterm = 4, false if dsterm != 4. How do we code this in the parser?

Reply:
can use `values = { 4 = true, 1 = false, 3 = false, 2 = false }`, where 1--4 are dsterm values